packages/contracts-bedrock/scripts/libraries/DeployUtils.sol

```diff
@@ -217,11 +219,28 @@ library DeployUtils {
 
     /// @notice Asserts that the given proxy has an implementation set.
     /// @param _proxy Proxy to check.
-    function assertImplementationSet(address _proxy) internal {
+    function assertERC1967ImplementationSet(address _proxy) internal returns (address implementation) {
```

Comment:
Originally has this function generic e.g. `assertImplementationSet` but it got messy. I ended up having to pass through 4 params i.e. `addressManager, proxy, proxyType, implementationName`. 

Splitting them was the cleaner approach.